package-lock.json

```diff
@@ -1,6195 +1,8 @@
 {
   "name": "@automattic/vip-go-mu-plugins",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
`lockfileVersion` speaks to format of the lock file itself, correct? So this would be happening because of a BC break in `lockfileVersion` between npm versions?

Im commenting based my perception of that being correct: if its breaking based on lock file version what we should do is put all developers on the same version of npm+ rather than juggle the lockfileVersion in package-lock.json. Whichever is the highest `lockfileVersion`; I think we should base our npm/node requirements around that.

Reply:
just thinking about it a little more, is it that the parameter: `lockfileVersion` has become defunct in the newest version of npm? I suppose that could also be a possibility. Im trying to find out. Can you comment on which node/npm you are running?

Reply:
```
jessegreathouse$ npm -v && node -v
8.0.0
v16.3.0
```